app/models/camaleon_cms/custom_field_group.rb

```diff
@@ -1,29 +1,41 @@
 class CamaleonCms::CustomFieldGroup < CamaleonCms::CustomField
   self.primary_key = :id
+
   # attrs required: name, slug, description
   alias_attribute :site_id, :parent_id
-  default_scope { where.not(object_class: '_fields').reorder("#{CamaleonCms::CustomField.table_name}.field_order ASC") }
 
-  has_many :metas, ->{ where(object_class: 'CustomFieldGroup')}, :class_name => "CamaleonCms::Meta", foreign_key: :objectid, dependent: :destroy
-  has_many :fields, -> {where(object_class: '_fields')}, :class_name => "CamaleonCms::CustomField", foreign_key: :parent_id, dependent: :destroy
-  belongs_to :site, :class_name => "CamaleonCms::Site", foreign_key: :parent_id
+  default_scope {
+    where.not(object_class: '_fields')
+    .reorder("#{CamaleonCms::CustomField.table_name}.field_order ASC")
```

Comment:
Not a fan of this style of line splitting in JS, did not realize it even worked in Ruby.